rosdep/base.yaml

```
@@ -4533,6 +4533,7 @@ libopencv-dev:
   rhel: [opencv-devel]
   ubuntu: [libopencv-dev]
 libopencv-imgproc-dev:
+  alpine: [opencv, opencv-dev]
```

`opencv-dev` has dependency to `opencv`, so
```suggestion
  alpine: [opencv-dev]
```
should be enough